### PR TITLE
Added commission screen

### PR DESCRIPTION
**Pull request by himanshu1365**
- Added commission screen
- Used common formatter in angular
- Changed double quotes with single quotes

### CODE REVIEW

File: frontend/uv-poc/src/app/service/consultant.service.ts [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 })
 export class ConsultantService {
   constructor(private http: HttpClient) {}
   baseUri: string = environment.baseUrl;
   get(consultantId) {
     let params = new HttpParams();
-    params = params.append("transactionId", consultantId);
-    return this.http.get(this.baseUri + "/consultant", {
+    params = params.append('transactionId', consultantId);
+    return this.http.get(this.baseUri + '/consultant', {
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/service/consultant.service.ts]: Make use of template literals, similar to what you've done in `frontend/uv-poc/src/app/service/commission.service.ts` line 17

File: frontend/uv-poc/src/index.html [hunk 2]
```diff
@@ -8,6 +8,8 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+  integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/index.html:11]: Kindly quote where you've used bootstrap

File: frontend/uv-poc/src/app/commission-details/commission-details.component.html [hunk 3]
```diff
@@ -0,0 +1,101 @@
+<div *ngIf="errorMsg; else commissionDataTable">
+  <div class="errorMsg">{{ errorMsg }}</div>
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/commission-details/commission-details.component.html]: What purpose does the `errorMsg` class fulfill here?

File: frontend/uv-poc/src/styles.css [hunk 4]
```diff
@@ -1,4 +1,12 @@
...
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+body {
+    margin: 0;
+    font-family: Roboto,"Helvetica Neue", sans-serif; 
+}
+
+table{
+    width: 100%;
+}
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/styles.css]: Consider using a material class to get the desired result

File: frontend/uv-poc/src/styles.css [hunk 5]
```diff
@@ -1,4 +1,12 @@
-/* You can add global styles to this file, and also import other style files */
+html, body { 
+    height: 100%; 
+}
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/styles.css]: Consider using a material class to get the desired result

**Review by KartikyaThakur — CHANGES_REQUESTED** [review]

**Comment by himanshu1365** [comment]
I have used bootstrap in commission.component.html file

File: frontend/uv-poc/src/app/app.module.ts [hunk 6]
```diff
@@ -13,13 +14,16 @@ import { ConsultantDetailsComponent } from './consultant-details/consultant-deta
...
     CustomerHistoryComponent,
   ],
   imports: [
     BrowserModule,
     AppRoutingModule,
     BrowserAnimationsModule,
+    MatTableModule,
+    HttpClientModule,
     MatPaginatorModule,
     MatTableModule,
```
Comment (KartikyaThakur) [inline frontend/uv-poc/src/app/app.module.ts]: Please spot the issue here.

**Pull request merged**